README.md

```diff
@@ -1,12 +1,8 @@
 ![conduit][logo]
 
+[![Build Status][ci-badge][ci]
```

Comment:
You're missing a closing bracket here. I think it should be:

```
[![Build Status][ci-badge]][ci]
```

![image](https://user-images.githubusercontent.com/9226/33858470-95e04ae8-de84-11e7-8313-9dcad5f1b73f.png)